app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -248,20 +248,25 @@ class LuciBuildService {
   ///
   /// Builds are queried based on the [RepositorySlug] and pull request number.
   Future<void> cancelBuilds(github.PullRequest pullRequest, String reason) async {
+    log.info(
+        'Attempting to cancel builds... for pullrequest ${pullRequest.base!.repo!.fullName}/${pullRequest.number}');
     final Map<String?, Build?> builds = await tryBuildsForPullRequest(pullRequest);
```

Comment:
is tryBuildsForPullRequest reporting builds with pending status?

Reply:
It is not specifying a state when it queries for the builds.

Reply:
Okay we cannot get the status pending returned:
![image](https://user-images.githubusercontent.com/32242716/230227266-ec8dd45f-aa96-4ffb-b51c-c7ef0893d14c.png)